packages/core/test/npm.js

```diff
@@ -29,45 +27,36 @@ describe("NPM integration", function() {
     };
     config.network = "development";
 
-    fs.writeFileSync(
+    fse.writeFileSync(
       path.join(config.contracts_directory, "Parent.sol"),
       parentContractSource,
       { encoding: "utf8" }
     );
   });
 
-  before("Create a fake npm source", function(done) {
-    var fake_source_path = path.join(
+  before("Create a fake npm source", () => {
+    const fakeSourcePath = path.join(
       config.working_directory,
       "node_modules",
       "fake_source",
       "contracts"
     );
 
-    async.series(
-      [
-        fse.ensureDir.bind(fse.ensureDir, fake_source_path),
-        fs.writeFile.bind(
-          fs,
-          path.join(fake_source_path, "Module.sol"),
-          moduleSource,
-          { encoding: "utf8" }
-        ),
-        fs.writeFile.bind(
-          fs,
-          path.join(fake_source_path, "ModuleDependency.sol"),
-          moduleDependencySource,
-          { encoding: "utf8" }
-        )
-      ],
-      done
+    fse.ensureDirSync(fakeSourcePath);
+    fse.writeFileSync(path.join(fakeSourcePath, "Module.sol"), moduleSource, {
+      encoding: "utf8"
+    });
+    fse.writeFileSync(
+      path.join(fakeSourcePath, "ModuleDependency.sol"),
+      moduleDependencySource,
+      { encoding: "utf8" }
```

Comment:
This is just a test so I wonder if it's necessary to remove async here since it can be a `devDep`?  Also, if we do choose to remove it do we want to fall back to synchronous operations?

Reply:
I say remove it so we don't have to think about it any more. I don't think it will matter much in this case. I don't really think we will lose much making this synchronous.

Reply:
Fair enough!

Both operations depend on ensureDir so making that synchronous makes sense.  It would be easy enough to have both writes be asynchronous though.  Gains are negligible plus this is a test.